src/node_modules/app/elasticsearch/config.js

```diff
@@ -335,8 +335,9 @@ export const defaultValues = {
 		order: {"_count":"desc"} // ES default
 	},
 	date_histogram: {
-		time_zone: 'UTC', // ES default
+		min_doc_count: 1, // Dapsboard default
```

Comment:
minor, eventually leaven these out for the defaultr PR

Reply:
You want `UTC` out?

Reply:
no, sorry I meant all defaults to be moved in the upcoming defaults PR for consistency, I should have selected all the relevant lines

Reply:
Ahh got it.